src/rpc/src/chainStorageWatcher.ts

```diff
@@ -0,0 +1,167 @@
+import { AgoricChainStoragePathKind } from './types';
+import { batchVstorageQuery } from './batchQuery';
+import type { UpdateHandler } from './types';
+import type { Unserialize } from '@endo/marshal';
+
+type Subscriber<T> = {
+  onUpdate: UpdateHandler<T>;
+};
+
+const defaultNewPathQueryDelayMs = 20;
+const defaultRefreshLowerBoundMs = 2000;
+const defaultRefreshUpperBoundMs = 4000;
```

Comment:
glad to see these constants. consider the following to match the strings of the params:
```suggestion
const defaults = {
  newPathQueryDelayMs: 20,
  refreshLowerBoundMs: 2000,
  refreshUpperBoundMs: 4000,
}
```